app/src/main/java/com/github/gogiffy/fragments/GifGridFragment.java

```diff
@@ -87,7 +87,8 @@ public void onLoadMore(int page, int totalItemsCount) {
         });
         loadTrendingGifs(0);
     }
-
+    public void testThisPR(){
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1186)